src/site/content/en/blog/one-line-layouts/index.md

```diff
@@ -0,0 +1,311 @@
+---
+title: Ten modern layouts in one line of CSS
+subhead: This post highlights a few powerful lines of CSS that do some serious heavy lifting and help you build robust modern layouts.
+authors:
+  - una
+description: This post highlights a few powerful lines of CSS that do some serious heavy lifting and help you build robust modern layouts.
+date: 2020-07-02
+hero: hero.jpg
+hero_position: top
+alt: Holy grail layout.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - css
+---
+
+
+This post will highlight a few powerful lines of CSS that do some serious heavy lifting and help you build robust modern layouts.
+
+-- video embed --
+
+I made this handy little site called 1 line layouts, at [1linelayouts.glitch.me](1linelayouts.glitch.me) so that you can follow along, or play on your own and have a reference for the power that CSS can bring to your layouts.
+
+## 01. Super Centered: `place-items: center`
+
+<figure class='w-figure'>
+<video controls autoplay loop muted class='w-screenshot'>
+    <source src='https://storage.googleapis.com/web-dev-assets/one-line-layouts/01-place-items-center.mp4'>
+  </video>
+</figure>
+
+For the  first 'single-line' layout, let's solve the biggest mystery in all of the CSS land: centering things. I want you to know that it's easier than you think with [`place-items: center`](https://developer.mozilla.org/en-US/docs/Web/CSS/place-items).
+
+First specify 'grid' as the display method, and then write `place-items: center` on the same element that you set  display to grid. `'Place-items'` is a shorthand to set the two both `'align-items'` and `'justify-items'` at once. By setting it to `'center'`, both `'align-items'` and `'justify-items'` are set to  `'center'`.
+
+```css/2
+.parent {
+  display: grid;
+  place-items: center;
+}
+```
+
+This enables the content to be perfectly centered within the parent, regardless of intrinsic size.
+
+## 02. The Deconstructed Pancake: `flex: <grow> <shrink> <baseWidth>`
+
+<figure class='w-figure'>
+<video controls autoplay loop muted class='w-screenshot'>
+    <source src='https://storage.googleapis.com/web-dev-assets/one-line-layouts/02-deconstructed-pancake-1.mp4'>
+  </video>
+</figure>
+
+Next we have the deconstructed pancake! This is a common layout for marketing sites, for example, which may have a row of 3 items, usually with an image, title, and then some text, describing some features of a product. On mobile, we'll want those to stack nicely, and expand as we increase the screen size. 
+
+By using flexbox for this effect, you won't need media queries to adjust the placement of these elements when the screen resizes.
```

Comment:
```suggestion
By using Flexbox for this effect, you won't need media queries to adjust the placement of these elements when the screen resizes.
```
Please format it as uppercase throughout the post so that we align with MDN's formatting: https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Flexbox